<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="radioBtn1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>92</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사과</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>92</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>수박</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>92</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>참외</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>92</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>복숭아</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
